src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,116 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY;
+    private static final float LOAD_FACTOR;
+    private static final int RESIZE_FACTOR;
+    private int currentCapacity;
+    private int size;
+    private int threshold;
+    private Node<K, V>[] table;
+
+    static {
+        INITIAL_CAPACITY = 16;
+        LOAD_FACTOR = 0.75f;
+        RESIZE_FACTOR = 2;
+    }
```

Comment:
you can initialize constants as soon as they are declared. this will visually shorten the length of the code

Reply:
Disagree. Once I did like this, and got mentor's instruction to initialize variables in initialization block.